README.md

```diff
@@ -34,8 +34,8 @@ python3 -m venv /path/to/new/virtual/environment
 source /path-to-new-virtual-environment/bin-activate
 ```
 
-Now install the requirements for this project: `pip install -r requirements.txt`
+Now install the requirements for this project: `pip3 install -r requirements.txt`
```

Comment:
This isn't right - if the virtual environment is set up with python3, then the version of pip should be correct